lib/css/exports/_stacks-constants-colors.less

```diff
@@ -106,44 +106,50 @@
 @red-200: #f4b4bb;
 @red-300: #e87c87;
 @red-400: #de535e;
-@red-500: #d1383d;
+@red-500: @red;
 @red-600: #c02d2e;
 @red-700: #ac2726;
 @red-800: #942121;
 @red-900: #7a1819;
 
 //  $  BADGES
 //  ----------------------------------------------------------------------------
-@badge-gold:    #ffcc01;
```

Comment:
This is a breaking change. Double check instances of this before updating any clients

Reply:
I'll make sure this rename goes into the production PR.